src/OrbitGl/CaptureWindowTest.cpp

```diff
@@ -302,6 +308,55 @@ TEST_F(NavigationTestCaptureWindow, PanTimeWorksAsExpected) {
   PreRender();
   EXPECT_EQ(time_graph_->GetHorizontalSlider()->GetPosRatio(), 0.0);
   EXPECT_EQ(time_graph_->GetMinTimeUs(), 0.0);
+
+  // Right arrow
+  KeyPressed(20, false, false, false);
+  PreRender();
+  EXPECT_GT(time_graph_->GetHorizontalSlider()->GetPosRatio(), 0.0);
+  EXPECT_GT(time_graph_->GetMinTimeUs(), 0.0);
+
+  // Left Arrow
+  KeyPressed(18, false, false, false);
+  PreRender();
+  EXPECT_EQ(time_graph_->GetHorizontalSlider()->GetPosRatio(), 0.0);
+  EXPECT_EQ(time_graph_->GetMinTimeUs(), 0.0);
+}
+
+TEST_F(NavigationTestCaptureWindow, Scrolling) {
+  PreRender();
+  ExpectInitialState();
+
+  int x = 0;
+  int y = viewport_.GetScreenHeight() - kBottomSafetyMargin;
+
+  // Mouse Wheel should scroll up and down.
+  MouseWheelMoved(x, y, -1, /*ctrl=*/false);
+  PreRender();
+  ExpectScrolledDownFromInitialState();
+
+  MouseWheelMoved(x, y, 1, /*ctrl=*/false);
+  PreRender();
+  ExpectInitialState();
+
+  // Down
+  KeyPressed(21, false, false, false);
```

Comment:
Nit: Constants for the keys would make this even easier to read. Here and below.